source/sgp_mode/DiversityAnalysis.h

```diff
@@ -60,6 +44,7 @@ emp::BitSet<64> ReturnTasksDone(CPU org_cpu) {
  */
 emp::unordered_map<emp::BitSet<64>, int> GetPhenotypeMap(emp::vector<CPU> organisms){
     emp::unordered_map<emp::BitSet<64> , int> phenotype_counts;
+    std::cout << "size: " << organisms.size()<< std::endl;
```

Comment:
Remember to remove debugging print statements before you submit to a pull request